modules/backup-restore/pages/enterprise-backup-restore.adoc

```diff
@@ -212,4 +212,16 @@ Unless otherwise specified, backup and restore apply both to _local_ and to _clo
 
 |===
 
-&#42; For local backup only -- _not_ for cloud.
++*+ For local backup only — _not_ for cloud.
+
+.Restoring metadata and users
+****
+* when restoring metadata to a newer Server version,
+if the feature that the metadata applies to no longer exists in the newer Server version, then the metadata may not be restorable.
+
+* If the user roles no longer exist in the version that you wish to restore to, then an error will be logged for the target user.
+
+* In general, if you can upgrade _directly_ to the new version, then you should be able to restore the users.
+If you cannot upgrade directly, then restoring users may cause errors.
```

Comment:
Maybe clarify the last sentence by adding, "for example, if some user roles no longer exist in the newer Server version."

If you cannot upgrade directly, then restoring users may cause errors -- for example, if some user roles no longer exist in the newer Server version.